src/elements/content-sidebar/__tests__/SidebarNavSign.test.tsx

```diff
@@ -1,80 +1,75 @@
 import * as React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
 import SidebarNavSign from '../SidebarNavSign';
 // @ts-ignore Module is written in Flow
 import FeatureProvider from '../../common/feature-checking/FeatureProvider';
+// @ts-ignore Module is written in Flow
+import messages from '../messages';
+// @ts-ignore Module is written in Flow
+import localize from '../../../../test/support/i18n.js';
+
+jest.unmock('react-intl');
 
 describe('elements/content-sidebar/SidebarNavSign', () => {
     const onClickRequestSignature = jest.fn();
     const onClickSignMyself = jest.fn();
+    const signLabel = localize(messages.boxSignRequestSignature.id);
+    const requestSignatureButtonText = localize(messages.boxSignRequestSignature.id);
+    const signMyselfButtonText = localize(messages.boxSignSignMyself.id);
```

Comment:
we should avoid using the messages directly in the assertion, if someone accidentally changes the messages, there is no way to verify if the change was intentional, we should be hardcoding the strings